### PR TITLE
Badness

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #2141

### CODE REVIEW

File: src/core/exporter.js [hunk 1]
```diff
@@ -86,6 +86,8 @@ function cleanup(cloneDoc) {
 
   insertions.appendChild(metaGenerator);
   head.prepend(insertions);
+  console.log("BEFORE SAVE");
+  debugger;
```
Comment (marcoscaceres) [inline src/core/exporter.js]: Stepping here will jump into pubsubhub... and checking `subscriptions` shows that that all the subscriptions have vanished :(

Comment (marcoscaceres) [reply src/core/exporter.js]: I think for whatever reason, this `pubsubhub` export must be different from the document's one... but I honestly don't know.

Comment (saschanaz) [reply src/core/exporter.js]: image

I can see them alive? 🤔

Comment (marcoscaceres) [reply src/core/exporter.js]: Seems to happen in the test harness. If you run Karma, the test fails (for me at least).

**Comment by marcoscaceres** [comment]
No way???? What was the bug???

**Comment by saschanaz** [comment]
I replaced `require()` with `doc.defaultView.require()`. I thought "What would it even mean when the function is from the parent document and the actual document element is from an iframe?", but I'm still trying to understand why it fixes the problem 🤔

**Comment by marcoscaceres** [comment]
Different realms?

**Comment by marcoscaceres** [comment]
Also, if you agree, I’d like to get this out as it’s going to affect the ability for people to automatically publish at the w3c.

**Comment by saschanaz** [comment]
I think I got it.

1. The modules from the parent document are the raw modules that haven't `run()`, so naturally its `pubhubsub` will get no subscriptions.
2. The previous code was `pub()`ing to the parent document.
3. So nothing happens 😁

**Comment by marcoscaceres** [comment]
Ok, that makes sense. Really appreciate you digging into this!

**Pull request merged**